proposals/array_modify.json

```diff
@@ -181,6 +161,22 @@
                 "b",
                 "c"
             ]
+        },
+        {
+            "description": "Remove multiple values from the end of the array and ignore that the given length is exceeding the size of the array.",
```

Comment:
Not sure whether this is a good example. It's more a unit test than a valuable example to users, so we can also remove it if desired.

Reply:
I'm fine with this example it illustrates removing with inserting and the clipping feature